gateway/src/main/java/org/georchestra/gateway/app/GeorchestraGatewayApplication.java

```diff
@@ -108,7 +110,10 @@ public String loginPage(Model mdl) {
         mdl.addAttribute("footer_url", georchestraFooterUrl);
         mdl.addAttribute("ldapEnabled", ldapEnabled);
         mdl.addAttribute("oauth2LoginLinks", oauth2LoginLinks);
-
+        if (allRequestParams.get("error") != null) {
+            if (allRequestParams.get("error").equals("expired_password"))
+                mdl.addAttribute("passwordExpired", true);
+        }
```

Comment:
this looks a bit bloated, how about
```
 if ("expired_password".equals(allRequestParams.get("error")))
     mdl.addAttribute("passwordExpired", true);
```
or just
```
   boolean expired = "expired_password".equals(allRequestParams.get("error"));
   mdl.addAttribute("passwordExpired", expired);
```

Reply:
@groldan  code updated and commits squashed. Thanks